wikimedia/wikipedia-ios
# Fix `mark as read` for notifications from MediaWiki

**Phabricator:** https://phabricator.wikimedia.org/T300320

### Notes
* Our calls to "mark as read" on MediaWiki API were not being effective, due to the resource being moved from `mediawiki.org` to `www.mediawiki.org`

### Test Steps
1. Generate some notifications from MediaWiki.
2. On the app, tap on the MediaWiki notification. Go back to the notification center and verify that is marked as read. 
3. Go to the website and verify if it was marked as read as well.
4. On the app, swipe left on the MediaWiki notification to mark it as unread. 
5. Close the notifications center and open it again. Check if the notification is marked as unread.
5. On the web version, check if the notification is now unread.
